### ISSUE TITLE
ensure controlnet respects the (canny) input image

### ISSUE THREAD

**Issue by benswift** [ANUcybernetics/imaginative-restoration#9]
I know that #2 is closed, but while it did work (no errors) the controlnet/adapter still doesn't really _work_. The output image has some resemblance to the input (cannified camera image), but it's pretty loose, and only when the step count is much higher than we need for real-time interaction.

Some ideas here:

- figure out how to get some python code out of Charlotte's comfyui workflow (which _does_ work) and try that
- try other controlnets (e.g. one designed specifically for SDXL turbo)
- double-check that SDXL Turbo is even working properly (it _seems_ to be, but it'd be good to have a definitive test by e.g. comparing it with the same prompt & seed to a generated image on another platform)

Basically, this issue is the _make it work_ part of the _make it work, then make it fast_ workflow. And as such, it's important 🙂 .
